pkg/generator_test.go

```diff
@@ -413,19 +413,19 @@ func newMockRequester_unexported(t testing.TB) *mockRequester_unexported {
 }
 
 func (s *GeneratorSuite) TestGeneratorExportedOnFlag() {
-	expected := `// Mockrequester_unexported is an autogenerated mock type for the requester_unexported type
-type Mockrequester_unexported struct {
+	expected := `// MockRequester_unexported is an autogenerated mock type for the requester_unexported type
+type MockRequester_unexported struct {
 	mock.Mock
 }
 
 // Get provides a mock function with given fields:
-func (_m *Mockrequester_unexported) Get() {
+func (_m *MockRequester_unexported) Get() {
 	_m.Called()
 }
 
-// NewMockrequester_unexported creates a new instance of Mockrequester_unexported. It also registers a cleanup function to assert the mocks expectations.
-func NewMockrequester_unexported(t testing.TB) *Mockrequester_unexported {
-	mock := &Mockrequester_unexported{}
+// NewMockRequester_unexported creates a new instance of MockRequester_unexported. It also registers a cleanup function to assert the mocks expectations.
+func NewMockRequester_unexported(t testing.TB) *MockRequester_unexported {
+	mock := &MockRequester_unexported{}
```

Comment:
Yeah, I thought so, no problem. But please, keep that in mind for v3, I think this should be fixed.

Reply:
Okey, so, should I add the breaking change back? :D I think it would be better to merge this now (it's non-BC), and I'll post another PR for v3...what do you think?

Reply:
I see yeah we can get this merged now since you reverted the breaking change. Thanks!